core/services/ocr2/plugins/mercury/integration_test.go

```diff
@@ -67,7 +67,7 @@ var (
 	}
 	rawReportingPluginConfig = relaymercury.OffchainConfig{
 		ExpirationWindow: 1,
-		BaseUSDFeeCents:  100,
+		BaseUSDFee:       decimal.NewFromInt(100),
```

Comment:
Is this right? IIUC @samsondav may have just changed the scale of these units

Reply:
Or probably irrelevant since this is only a test. Surprised it didn't affect production code though :thinking:

Reply:
All tests are green now ;)